### PR TITLE
detecting focus on devtools webcontents

### PR DESCRIPTION
**Pull request by deepak1556**
Is there a better way to communicate the event ?

### CODE REVIEW

File: browser/inspectable_web_contents_impl.cc [hunk 1]
```diff
@@ -241,6 +241,11 @@ void InspectableWebContentsImpl::AppendToFile(
     delegate_->DevToolsAppendToFile(url, content);
 }
 
+void InspectableWebContentsImpl::WebContentsFocused(
+    content::WebContents* contents) {
+    delegate_->DevToolsFocused();
```
Comment (zcbenz) [inline browser/inspectable_web_contents_impl.cc]: Needs to test whether `delegate_` is null here.

**Comment by zcbenz** [comment]
I think current way is good.

File: browser/inspectable_web_contents_impl.cc (hunk 1 above)
Comment (deepak1556) [inline browser/inspectable_web_contents_impl.cc]: Fixed, thanks!

**Comment by zcbenz** [comment]
:+1:
